content/docs/user-guide/data-management/index.md

```diff
@@ -36,4 +36,4 @@ DVC remotes are similar to Git remotes, but for <abbr>cached</abbr> data.
 
 </admon>
 
-[dvc remotes]: /doc/command-reference/remote
+[dvc remotes]: /doc/user-guide/data-management/remote-storage
```

Comment:
Another link update.

- [ ] Many more of these may be needed in the future but it could be a follow-up PR.

Reply:
> > Another link update.
>
> Many more of these may be needed in the future 

Being done in https://github.com/iterative/dvc.org/pull/4260 and https://github.com/iterative/dvc.org/pull/4262.